molecule/commands.py

```diff
@@ -159,6 +159,8 @@ def converge(self, idempotent=False, create_instances=True, create_inventory=Tru
             kwargs.pop('_err', None)
             kwargs['_env']['ANSIBLE_NOCOLOR'] = 'true'
             kwargs['_env']['ANSIBLE_FORCE_COLOR'] = 'false'
+            kwargs['_env']['ANSIBLE_CALLBACK_PLUGINS'] = os.path.join(
+                sys.prefix, 'share/molecule/ansible/plugins/callback/idempotence')
```

Comment:
It could be set as a default in ansible.cfg.j2 as a raw_env_var under the ansible block and then overridden by a user if they set their own value for that environment var in their molecule.yml.  The idempotence command could check to see if the plugin is available, and if not, print a warning about falling back to the method of parsing the output and not displaying the tasks.